src/core/client/framework/lib/validation.tsx

```diff
@@ -106,6 +107,17 @@ export const validateURL = createValidator(
   INVALID_URL()
 );
 
+/**
+ * validateURLStart is a Validator that checks that the URL starts with 'http'.
+ */
+export const validateURLStart = createValidator((v) => {
+  if (!startsWith(v, "http")) {
+    return false;
+  }
+
+  return true;
+}, INVALID_URL_START());
+
```

Comment:
It should check `startsWith(v, "http://") || startsWith(v, "https://")` instead and the method should maybe named `validateURLHTTP`.